HotelManagement/src/main/java/eu/deltasource/internship/hotelmanagement/core/HotelServiceApplication.java

```diff
@@ -0,0 +1,50 @@
+package eu.deltasource.internship.hotelmanagement.core;
+
+import eu.deltasource.internship.hotelmanagement.core.commodities.Bed;
+import eu.deltasource.internship.hotelmanagement.core.commodities.Shower;
```

Comment:
eu.deltasource.internship.hotelmanagement.core.commodities.*; 
will import everything in that package;